src/components/Navbar/ProfileComponents.tsx

```diff
@@ -85,7 +98,7 @@ const ProfileComponents = () => {
         <Image
           className="cursor-pointer z-[1] rounded-full"
           onClick={() => router.push("/profile")}
-          src={data.user.image}
+          src={user?user.image:data?.user?.image}
```

Comment:
This is took me a couple of trys to fully understand, all the question marks really do hurt the readability.


```js
if (user) {
  src = user.image
} else {
  /**
   * get data from next auth: useSession().data.user.image
   * 
   * but since pretty much each part of the useSession can be undefined
   * use the quetion mark operator to avoid throwing errors
   */
  src = data?.user?.image
}
```